GoMeet/src/main/webapp/create-meeting-step1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <!-- Create Meeting Step 1: Set meeting name and add guests -->
+  <head>
+    <title>GoMeet Create Meeting Step 1</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="js/create-meeting-step1.js"></script>
+  </head>
+  <body onload="addGuest(document);">
+    <h1 style="text-align:center;">GoMeet</h1>
```

Comment:
nit: style should be in the style sheet :https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.

You could consider adding a '.center' class to assign to these h1's and other elements you want to center.

Reply:
Done :)